src/loaders/gtdb/gtdb_taxa_count_loader.py

```diff
@@ -8,7 +8,6 @@
 import src.common.storage.collection_and_field_names as names
 from src.service.data_products import taxa_count
 
-# TODO DATAPROD see if there's a way to write the collection names into the jsonl
```

Comment:
There doesn't seem to be. Tried with `_id` and no collection name in the arangoimport command line but arangoimport complained about the missing collection name.